tests/test_covar2d.py

```diff
@@ -88,7 +89,7 @@ def testGetCovarCTF(self):
     def testGetCovarCTFShrink(self):
         results = np.load(os.path.join(DATA_DIR, 'clean70SRibosome_cov2d_covarctf_shrink.npy'))
         covar_opt = {'shrinker': 'frobenius_norm', 'verbose': 0, 'max_iter': 250, 'iter_callback': [],
-                     'store_iterates': False, 'rel_tolerance': 1e-12, 'precision': 'float64'}
+                     'store_iterates': False, 'rel_tolerance': 1e-12, 'precision': 'float32'}
```

Comment:
Not `self.dtype`?

Reply:
Yeah, looking at it now I think I'll want to go through some of the opt dictionaries. Good catch, thanks.

Reply:
I hit all the _opt dictionaries that turned up with grep to use self.dtype (source code and tests). Thanks for pointing out, I over looked it.